api/specs.proto

```diff
@@ -258,6 +258,12 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+
+	message SecurityConfig {
+	    string userns = 1;
+	    string credential_spec = 2 [(gogoproto.customname) = "CredentialSpec"];
```

Comment:
It _should_ contain a string that is something like `file:///...` or `registry://` I believe (https://github.com/docker/docker/blob/c0a1d2e0d88ff3cae6802dfbd128c7739e8c2bcc/daemon/start_windows.go#L135).  @johnstep can you confirm?